refinery/analysis_manager/tasks.py

```diff
@@ -464,19 +464,8 @@ def _get_galaxy_download_task_ids(analysis):
 
     # retrieving list of files to download for workflow
     tool = _get_workflow_tool(analysis.uuid)
-    tool.create_workflow_file_downloads()
     tool.create_analysis_output_node_connections()
 
-    dl_files = analysis.workflow_dl_files
-    # creating dictionary based on files to download predetermined by workflow
-    # w/ keep operators
-    dl_dict = {}
```

Comment:
huh: This dict didn't seem to be used, even before your changes? Just vestigial, probably?

Reply:
@mccalluc It was used way back when we were launching analyses the old way, but looking at this code in particular was the motivator for the removal of the whole model